src/org/usfirst/frc/team2175/properties/JoystickProperties.java

```diff
@@ -27,13 +28,20 @@ protected void populate() {
         gamepadPort = getIntPropertyValue("joystick.gamepad.port");
 
         deadbandSize = getDoublePropertyValue("deadband.value");
-        runfeeder = getIntPropertyValue("button.feeder.out");
-        runshooter = getIntPropertyValue("button.shooter.out");
-        shiftButtonNumber = getIntPropertyValue("joystickbutton.shift");
-        gearIntakeInNumber = getIntPropertyValue("button.gearintake.in");
-        gearIntakeOutNumber = getIntPropertyValue("button.gearintake.out");
-        runshooterin = getIntPropertyValue("joystickbutton.shooter.in");
-        runfeederin = getIntPropertyValue("joystickbutton.feeder.in");
+
+        runFeederOutInfo = getInfo("button.feeder.out");
+        runShooterOutInfo = getInfo("button.shooter.out");
+        shiftButtonInfo = getInfo("button.shift");
+        gearIntakeInInfo = getInfo("button.gearintake.in");
+        gearIntakeOutInfo = getInfo("button.gearintake.out");
+        runShooterInInfo = getInfo("button.shooter.in");
+        runFeederInInfo = getInfo("button.feeder.in");
+    }
+
+    public String[] getInfo(String str) {
+        String[] data = getStringPropertyValue(str).split(",");
+        data[1] = data[1].trim();
+        return data;
     }
```

Comment:
Tests for what?
When testing it, I made sure it returned an integer, if that is what you are talking about..